docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -231,29 +231,34 @@ V&V Checks:
     -    
   * - 3
     - % of each facility type have cesarian section capabilities
-    - Public/governmental health facility (62%), private and/or NGO health facility (73%), at home  (0%), other (50%) 
+    - Public/governmental hospital (62%), private and/or NGO health facility (73%), at home  (0%), country-specific governmental health facility (50%) 
```

Comment:
It looks like here you still have the broader "private and/or NGO health facility" category rather than splitting it into "private for-profit health facility" and "private not-for-profit health facility". Was that intentional? It looks like the original change to this line in this PR was:
> Public/governmental health facility (62%), private for-profit health facility (60%), at home  (0%), private not-for-profit health facility (86%), other (10%)

Maybe there was an accidental reversion when you updated the category names?

Reply:
Ah, it looks like this might have been due to a merge conflict with the other PR.

Reply:
Oops, thanks for catching that! Should be fixed now.